components/script/dom/htmlelement.rs

```diff
@@ -151,7 +151,11 @@ impl HTMLElementMethods for HTMLElement {
     // https://html.spec.whatwg.org/multipage/#handler-onload
     fn GetOnload(&self) -> Option<Rc<EventHandlerNonNull>> {
         if self.is_body_or_frameset() {
-            window_from_node(self).GetOnload()
+            if document_from_node(self).has_browsing_context() {
+                window_from_node(self).GetOnload()
+            } else {
+                None
```

Comment:
I believe this is consistent with the following specification definition (mentioned in #15332):
> If an event handler IDL attribute exposes an event handler of an object that doesn't exist, it must always return null on getting and must do nothing on setting.